### ISSUE TITLE
batch size的选择

### ISSUE THREAD

**Issue by xxm1668** [mark1879/Baichuan-13B-Finetuning#4]
[INFO|trainer.py:1786] 2023-07-12 09:11:08,040 >> ***** Running training *****
[INFO|trainer.py:1787] 2023-07-12 09:11:08,040 >> Num examples = 5,623
[INFO|trainer.py:1788] 2023-07-12 09:11:08,040 >> Num Epochs = 50
[INFO|trainer.py:1789] 2023-07-12 09:11:08,040 >> Instantaneous batch size per device = 8
[INFO|trainer.py:1790] 2023-07-12 09:11:08,040 >> Total train batch size (w. parallel, distributed & accumulation) = 32
[INFO|trainer.py:1791] 2023-07-12 09:11:08,040 >> Gradient Accumulation steps = 4
[INFO|trainer.py:1792] 2023-07-12 09:11:08,040 >> Total optimization steps = 8,750
[INFO|trainer.py:1793] 2023-07-12 09:11:08,041 >> Number of trainable parameters = 6,553,600
{'loss': 2.8251, 'learning_rate': 5e-05, 'epoch': 0.06}
{'loss': 2.7261, 'learning_rate': 5e-05, 'epoch': 0.11}
{'loss': 2.6971, 'learning_rate': 4.999995971591123e-05, 'epoch': 0.17}
{'loss': 2.7379, 'learning_rate': 4.999963744397995e-05, 'epoch': 0.23}
{'loss': 2.752, 'learning_rate': 4.999899290427178e-05, 'epoch': 0.28}
{'loss': 2.7516, 'learning_rate': 4.9998026105095405e-05, 'epoch': 0.34}
{'loss': 2.77, 'learning_rate': 4.999673705891373e-05, 'epoch': 0.4}
{'loss': 2.7204, 'learning_rate': 4.999512578234374e-05, 'epoch': 0.46}
{'loss': 2.7422, 'learning_rate': 4.999319229615621e-05, 'epoch': 0.51}

震荡，怎么解决

**Comment by mark1879** [comment]
LLaMA-Efficient-Tuning项目的负责人在处理这个事情，这个项目是从LLaMA-Efficient-Tuning项目中精简出来的，只能先等他那边的消息。
